en/mozorg/mission.ftl

```diff
@@ -11,10 +11,18 @@ mission-our-mission-is-to-ensure-the-internet = Our mission is to ensure the Int
 # Only shown if mission-our-mission-is-to-ensure-the-internet is not translated
 mission-our-mission-is-to-promote-openness = Our mission is to promote openness, innovation & opportunity on the Web.
 
+mission-at-mozilla-were-a-global-community-v2 = At { -brand-name-mozilla }, we’re a global community of technologists, thinkers and builders working together to keep the Internet alive and accessible, so people worldwide can be informed contributors and creators of the Web.
```

Comment:
Should `Web` be lowercase `w`?

Reply:
I think we usually capitalize it as a proper noun when referring to "the Web"

Reply:
For what it's worth: https://mozilla-hub.atlassian.net/wiki/spaces/FIREFOX/pages/11048304/Word+List#WordList-web